src/NodeChanger/MethodNameChanger.php

```diff
@@ -3,20 +3,22 @@
 namespace Rector\NodeChanger;
 
 use PhpParser\Node;
-use PhpParser\Node\Expr\MethodCall;
 use PhpParser\Node\Identifier;
 
 final class MethodNameChanger
 {
+    public function renameNode(Node $node, string $newMethodName): void
```

Comment:
Is there some case when `MethodCall` typehint fails?

Reply:
Yep, see [this commit](https://github.com/rectorphp/rector/commit/4eb9c376dc25f953009831f9a1c5da1cad9c70a2), I explained why on its title

Reply:
This is exactly the case where this strict typing and decoupling to this service helps :)

It should be fixed on place where `Expr` is passed. `MethodCall` should be passed instead.

Could you change that on responsible place?